apps/api-reference/next.config.js

```diff
@@ -9,6 +9,18 @@ export default {
     },
   },
 
+  redirects: async () => [
+    {
+      source: "/home",
+      destination: "/",
+      permanent: true,
+    },
+    {
+      source: "/price-feeds/evm",
+      destination: "/price-feeds/evm/parsePriceFeedUpdates",
+      permanent: true,
```

Comment:
We shouldn't use `permanent` for either of these two redirects; a permanent redirect is cached by the browser and never re-checked (at least, while the browser considers the cache live, which is around 6 months for Chrome and the spec does not require browsers to cap at all).  As a result, doing this will prevent us from being able to add pages to these URLs in the future, if we ever wanted them to not be redirects.